servicemesh-install/topics/install-prerequisites.adoc

```diff
@@ -0,0 +1,252 @@
+|false
+
+|prefix
+|Any valid image repo
+|Which prefix to apply to Istio image names used in docker pull.
+|If deployment_type=origin the default is `maistra/`
+
+If deployment_type=openshift the default is `openshift-istio-tech-preview/` 
+
+|version
+|Any valid Docker tag
+|Docker tag to use with Istio images
+|0.1.0
+|===
+
+.Jaeger Parameters
+|===
+|Parameter |Values |Description  |Default
+
+|prefix
+|Any valid image repo
+|Which prefix to apply to Jaeger image names used in docker pull.
+|If deployment_type=origin the default is `maistra/`
+
+If deployment_type=openshift the default is `distributed-tracing-tech-preview/`
+
+|version
+|Any valid Docker tag
+|Which Docker tag to use with Jaeger images
+|If deployment_type=origin then 1.6
+
+If deployment_type=openshift then 1.6.0
+
+|elasticsearch_memory
+|Memory size in megabytes or gigabytes.
+|The amount of memory to allocate to the Elasticsearch installation
+|512M
+|===
+
+.Launcher Parameters
+|===
+|Component |Parameter |Description |Default
+
+|openshift
+|user
+|Should be modified to reflect the OpenShift user that you want to use to run the Fabric8 launcher.  
+|`developer`
+
+|
+|password
+|OpenShift user password for running Fabric8 launcher.
+|`developer`
+
+|github
+|username
+|Should be modified to reflect the GitHub user name for the https://help.github.com/articles/signing-up-for-a-new-github-account/[GitHub account] you want to use to run the Fabric8 launcher.  
+|N/A
+
+|
+|token
+|GitHub https://github.com/settings/tokens[personal access token] you want to use to run the Fabric8 launcher.  
+|N/A
+
+|catalog
+|filter
+|Red Hat booster catalog filter
+|booster.mission.metadata.istio
+
+|
+|branch
+|Red Hat Booster catalog to use for Fabric8
+|v35
+
+|
+|repo
+|GitHub repo to use for Fabric8
+|https://github.com/fabric8-launcher/launcher-booster-catalog.git
+|===
+
+
+== Creating the Operator Namespace
+
+NOTE: The custom resource *must* be deployed into the `istio-operator` namespace.
+
+. Create the `istio-operator` namespace with the following command: 
++
+```
+oc new-project istio-operator
+```
++
+. Verify the namespace was created: 
++
+```
+oc get projects
+```
+
+= Preparing the OCP 3.10 Installation
+
+Before the {ProductShortName} can be installed into an OCP 3.10 installation it is necessary to make a number of changes to the master configuration and each of the schedulable nodes.  These changes will enable features required within the {ProductShortName} and also ensure Elasticsearch will function correctly.
+
+== Updating the Master Configuration
+
+NOTE: The community version of Istio will inject the sidecar by default if you have labeled the namespace.  You are not required to label the namespace with {ProductName}. However, {ProductName} requires you to opt-in to having the sidecar automatically injected to a deployment.  This is to avoid injecting a sidecar where it is not wanted (for example build or deploy pods).
```

Comment:
I'm not marking it up anymore, but please scrub the future tense.
What do you need to label the namespace to get the sidecar?

Reply:
You don't need to label the namespace.